Lesson_16/l16_h01_class.py

```diff
@@ -0,0 +1,77 @@
+"""
+HW:16[Volkov Anton].
+
+# write class for bank card.
+# Class should reflect card lifecycle, card operations etc.
+# use CVV, PIN, Name, Surname , end date, card_num as initial params
+# class should have in addition to common logic some class attributes,
+# as minimum one classmethod and
+# as minimum  one staticmethod, two or more getters/setters
+# do not forget about block if __name__ == '__main__':
+"""
+
+
+class MonoBankCard:
+    """Class for creating cards."""
+
+    _lvl_card = None
```

Comment:
Pay attention that class attribute is something that will be the same for all class instances. For example here could be PIN_MAX_SIZE = 4 or BANK_ID = 5375 or  smth like this . Card lvl is something unique for certain instance - thus we should use it as instance attribute , but not class attribute.